app/src/main/java/com/kyphipraniti/fitnesstasks/fragments/CalendarFragment.java

```diff
@@ -83,6 +86,10 @@
     public CalendarFragment() {
     }
 
+    public static List<Task> getmCompletedTasks() {
```

Comment:
this should either be `getMCompletedTasks` to keep with camel case conventions or `getCompletedTasks`. I like the latter, as it abstracts knowledge of how we actually store the tasks in a variable from other functions that have to call this